ReactiveObjCBridge/RACScheduler+SwiftSupport.h

```diff
@@ -0,0 +1,5 @@
+#import <ReactiveObjC/ReactiveObjC.h>
+
+@interface RACScheduler (SwiftSupport)
++ (RACScheduler *) schedulerWithRACSwiftScheduler:(RACScheduler*)scheduler;
```

Comment:
I think this file needs to be added to the umbrella header.

Edit: Now I accidentally started a review. Oops.